### DISCUSSION TITLE
1.2.4 no value found for choices

### DISCUSSION THREAD

**Discussion by t8or** [Yue-Yang/ChatGPT-Siri#91] [General]
![IMG_4355](https://user-images.githubusercontent.com/17255992/227841727-40657741-655b-4be9-b1b1-9fd7d0e09c28.jpeg)

Any idea what’s not working? On iOS 16.4 beta

**Comment by sbdk** [comment]
> 

`gpt-4` model seems not working, tested with other API tools.
use `gpt-3.5-turbo` will do the trick, works as expected.

**Comment by dev-brendanprice** [comment]
> > 
> 
> `gpt-4` model seems not working, tested with other API tools. use `gpt-3.5-turbo` will do the trick, works as expected.

Tested this on version(s) `1.2.3` and `1.2.4`, each on `gpt-3.5-turbo` and `gpt-4`. Still says the same error.
iOS version 16.0

**Comment by shawnyeager** [comment]
I’m seeing the same error.

**Comment by sbdk** [comment]
> > > 
> > 
> > 
> > `gpt-4` model seems not working, tested with other API tools. use `gpt-3.5-turbo` will do the trick, works as expected.
> 
> Tested this on version(s) `1.2.3` and `1.2.4`, each on `gpt-3.5-turbo` and `gpt-4`. Still says the same error. iOS version 16.0

make sure you have available credit to use for you OpenAi account. You can check that under you Account/Usage page.

**Comment by Yue-Yang** [comment]
# choices check this:
https://github.com/Yue-Yang/ChatGPT-Siri/discussions/30

# gpt-4
Currently, GPT-4 requires joining a waitlist. Once you join the waitlist, you will receive a confirmation email. Then, you need to wait for the invitation to use the GPT-4 API, and after you have qualified, you can see the GPT-4 model option for testing in the playground backend. If you do not have GPT-4 qualifications, please choose to use the GPT-3.5-Turbo model.

**Reply by Tempus02** [reply]
I am using GPT-3.5-Turbo, however im still facing the same problem.
